tools/container/toolkit/toolkit.go

```diff
@@ -793,3 +825,98 @@ func generateCDISpec(opts *Options, nvidiaCDIHookPath string) error {
 
 	return nil
 }
+
+// getRuntimeBinaryPaths extracts the full paths of the low-level runtime binaries specified in the container runtime config
+func getRuntimeBinaryPaths(hostRoot string) ([]string, error) {
+	var runtimeBinaryPaths []string
+
+	if r, ok := os.LookupEnv("RUNTIME"); ok {
```

Comment:
We should extract this as the top-level and pass this down to the toolkit logic. The toolkit should NOT read the environment variable to determine this value since there are multiple ways for this to be set in the CLI that is being invoked.

Reply:
This is done